index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div id="container">
+    <nav id="navigation">
+      <span id="logo">My Logo</span>
+      <ul id="menu">
+        <li class="menu-item big">Portfolio</li>
+        <li class="menu-item big">About</li>
+        <li class="menu-item big">Contact</li>
+        <li class="menu-item" id="hidden"><i class="fas fa-bars"></i></li>
+      </ul>
+    </nav>
```

Comment:
- [ ] I think you can structure this section in a better and more semantic way:
```
<header>
  <span>logo</span>
  <nav>
    <burger icon>
    <ul>...</ul>
  </nav>
</header>
```
This will ensure that only navigation elements are inside the `nav` tag, this is also good for accessibility! :slightly_smiling_face:

Reply:
Very good point.